### ISSUE TITLE
500 error on profile page in Firefox

### ISSUE THREAD

**Issue by g4s8** [zerocracy/farm#1080]
Profile page `/u/(username)` returns 500 error in Firefox, other pages (like `/agenda`, `/awards`) are working fine.
``` 
java.lang.IllegalStateException: [GET https://www.0crat.com/u/g4s8] failed in 508ms: net.sf.saxon.trans.XPathException: Cannot compare xs:untypedAtomic to xs:integer
	at org.takes.facets.fallback.TkFallback.error(TkFallback.java:234)
	at org.takes.facets.fallback.TkFallback.access$100(TkFallback.java:54)
	at org.takes.facets.fallback.TkFallback$2.body(TkFallback.java:202)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.rs.RsPrint.printBody(RsPrint.java:161)
	at org.takes.rs.RsPrint.print(RsPrint.java:114)
	at org.takes.http.BkBasic.print(BkBasic.java:108)
	at org.takes.http.BkBasic.accept(BkBasic.java:84)
	at org.takes.http.BkSafe$1.accept(BkSafe.java:51)
	at org.takes.http.BkWrap.accept(BkWrap.java:52)
	at org.takes.http.BkTimeable.accept(BkTimeable.java:89)
	at org.takes.http.BkParallel$1$1.run(BkParallel.java:89)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.io.IOException: net.sf.saxon.trans.XPathException: Cannot compare xs:untypedAtomic to xs:integer
	at org.takes.rs.RsXslt.transform(RsXslt.java:133)
	at org.takes.rs.RsXslt.access$100(RsXslt.java:89)
	at org.takes.rs.RsXslt$1.body(RsXslt.java:113)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.facets.fork.RsFork$1.body(RsFork.java:74)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at com.zerocracy.tk.TkSoftForward$RsSoft$1.body(TkSoftForward.java:95)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.facets.forward.TkForward$Safe.load(TkForward.java:116)
	at org.takes.facets.forward.TkForward$Safe.body(TkForward.java:102)
	at org.takes.rs.RsWithHeader$1.body(RsWithHeader.java:114)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.rs.RsGzip.make(RsGzip.java:91)
	at org.takes.rs.RsGzip.body(RsGzip.java:76)
	at org.takes.facets.fork.RsFork$1.body(RsFork.java:74)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.rs.RsWithHeader$1.body(RsWithHeader.java:114)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.rs.RsWithHeader$1.body(RsWithHeader.java:114)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.rs.RsWithHeaders$1.body(RsWithHeaders.java:78)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.facets.fallback.TkFallback$2.body(TkFallback.java:187)
	... 12 more
Caused by: net.sf.saxon.trans.XPathException: Cannot compare xs:untypedAtomic to xs:integer
	at net.sf.saxon.expr.ValueComparison.compare(ValueComparison.java:749)
	at net.sf.saxon.expr.ValueComparison.effectiveBooleanValue(ValueComparison.java:695)
	at net.sf.saxon.expr.instruct.Choose.choose(Choose.java:902)
	at net.sf.saxon.expr.instruct.Choose.processLeavingTail(Choose.java:877)
	at net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)
	at net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)
	at net.sf.saxon.expr.instruct.TemplateRule.applyLeavingTail(TemplateRule.java:347)
	at net.sf.saxon.trans.Mode.applyTemplates(Mode.java:502)
	at net.sf.saxon.expr.instruct.ApplyTemplates.apply(ApplyTemplates.java:295)
	at net.sf.saxon.expr.instruct.ApplyTemplates.processLeavingTail(ApplyTemplates.java:252)
	at net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)
	at net.sf.saxon.expr.instruct.TemplateRule.applyLeavingTail(TemplateRule.java:347)
	at net.sf.saxon.trans.Mode.applyTemplates(Mode.java:502)
	at net.sf.saxon.expr.instruct.ApplyTemplates.apply(ApplyTemplates.java:295)
	at net.sf.saxon.expr.instruct.ApplyTemplates.process(ApplyTemplates.java:248)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)
	at net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)
	at net.sf.saxon.expr.instruct.TemplateRule.applyLeavingTail(TemplateRule.java:347)
	at net.sf.saxon.trans.Mode.applyTemplates(Mode.java:502)
	at net.sf.saxon.expr.instruct.ApplyTemplates.apply(ApplyTemplates.java:295)
	at net.sf.saxon.expr.instruct.ApplyTemplates.processLeavingTail(ApplyTemplates.java:252)
	at net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)
	at net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)
	at net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)
	at net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)
	at net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)
	at net.sf.saxon.expr.instruct.TemplateRule.applyLeavingTail(TemplateRule.java:347)
	at net.sf.saxon.trans.Mode.applyTemplates(Mode.java:502)
	at net.sf.saxon.trans.TextOnlyCopyRuleSet.process(TextOnlyCopyRuleSet.java:66)
	at net.sf.saxon.trans.Mode.applyTemplates(Mode.java:478)
	at net.sf.saxon.Controller.transformDocument(Controller.java:2402)
	at net.sf.saxon.Controller.transform(Controller.java:1970)
	at net.sf.saxon.s9api.XsltTransformer.transform(XsltTransformer.java:596)
	at net.sf.saxon.jaxp.TransformerImpl.transform(TransformerImpl.java:73)
	at org.takes.rs.RsXslt.transform(RsXslt.java:160)
	at org.takes.rs.RsXslt.transform(RsXslt.java:131)
	... 36 more
```

**Comment by g4s8** [comment]
@0crat in

**Comment by 0crat** [comment]
> @0crat in ([here](https://github.com/zerocracy/farm/issues/1080#issuecomment-394119894))

@g4s8 Job [#1080](https://github.com/zerocracy/farm/issues/1080) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by g4s8** [comment]
@0crat assign @krzyk

**Comment by 0crat** [comment]
@krzyk[/z](https://www.0crat.com/u/krzyk) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by 0pdd** [comment]
@g4s8 the puzzle [#1142](https://github.com/zerocracy/farm/issues/1142) is still not solved.

**Comment by krzyk** [comment]
@g4s8 please close this issue, it has been merged in  #1131

**Comment by g4s8** [comment]
@krzyk thanks

**Comment by g4s8** [comment]
@rultor release, tag=`0.22.3`

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @krzyk[/z](https://www.0crat.com/u/krzyk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1080](https://github.com/zerocracy/farm/issues/1080) is now out of scope

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +35 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Comment by 0pdd** [comment]
@g4s8 the puzzle [#1399](https://github.com/zerocracy/farm/issues/1399) is still not solved; solved: [#1142](https://github.com/zerocracy/farm/issues/1142).
